care/facility/models/json_schema/daily_round.py

```diff
@@ -16,10 +16,16 @@
         {
             "type": "object",
             "properties": {
-                "name": {"type": "string"},
-                "quantity": {"type": "number"},
-                "concentration": {"type": "number"},
-                "conc_unit": {"type": "string"},
+                "name": {
+                    "enum": [
+                        "Adrenalin",
+                        "Nor-adrenalin",
```

Comment:
`Nor-adrenalin` is this a typo? or is it "No**n**-adrenalin"

Reply:
@aparnacoronasafe could you confirm?

<img width="733" alt="image" src="https://github.com/user-attachments/assets/4828891a-2816-4b84-98fb-d2e0181e4900">

Reply:
It should be "Noradrenaline". The "-" is not needed